lms/templates/instructor/instructor_dashboard_2/cohort-form.underscore

```diff
@@ -25,7 +25,7 @@
                             <%- gettext('Cohort Name') %> *
                             <span class="sr"><%- gettext('(Required Field)')%></span>
                     </label>
-                    <input name="cohort-name" value="<%= cohort_name_value %>" class="input cohort-name" id="cohort-name"
+                    <input name="cohort-name" value="<%- cohort_name_value %>" class="input cohort-name" id="cohort-name"
```

Comment:
Yes, that's accurate. I thought this was the correct approach, just wanted a second set of eyes to confirm.